client/state/products-list/selectors.js

```diff
@@ -1,3 +1,7 @@
+import { property } from 'lodash';
+
+export const getProductsList = property( 'productsList.items' );
```

Comment:
> what test data do I use?

What you expect your sub-tree to look like. I also find it useful to read tests to see what the general structure of what something looks like without having to spin up a branch.

>  I'm much more excited about testing behaviors.

Selectors are simple functions. We're just testing that we get the desired output for a given input. Ie what happens when an item exists, when happens when it doesn't, are we accidentally mutating data as we do this? 

Are you asking why we have tests in general?